taskcluster/scripts/pipeline/train_taskcluster.py

```diff
@@ -1,14 +1,101 @@
 #!/usr/bin/env python3
 
+import logging
+import os
 import os.path
+import requests
 import subprocess
 import sys
 
 TRAINING_SCRIPT = os.path.join(os.path.dirname(__file__), "train-taskcluster.sh")
+CONTINUATION_ARTIFACTS = {
+    "config.opustrainer.yml",
```

Comment:
@eu9ene - I originally used the list from https://github.com/mozilla/firefox-translations-training/blob/99015e13d50cb2b3222d916fefc71e78505639fe/taskcluster/translations_taskgraph/transforms/training_continuation.py#L5 for this, but found it didn't work due to `final` models being present. Please sanity check this list.

And tangentially related: should we update the training continuation artifact list to expect opustrainer files?

Reply:
Yes, to continue training from the same place OpusTrainer needs `config.opustrainer.yml.state` (this is a default path for the state file). I see that it already outputs this to the artifacts.

Ideally, we should modify the train.sh not to rebuild the config again if it's already present but this code is deterministic so it should work.